File: LJX_HorizontalPage/LJX_HorizontalPage/ViewController.m
//
//  ViewController.m
//  LJX_HorizontalPage
//
//  Created by 理享学 on 2021/9/2.
//

#import "ViewController.h"
#import "LJX_CollectionView.h"

@interface ViewController ()

@property(nonatomic, strong)LJX_CollectionView* baseView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
   
    [self createUI];
}

- (void)createUI {
    [self.view addSubview:self.baseView];
}

- (LJX_CollectionView *)baseView {
    if (!_baseView) {
        _baseView = [[LJX_CollectionView alloc] initWithFrame:self.view.bounds];
    }
    return _baseView;
}

@end

File: LJX_HorizontalPage_Swift/LJX_HorizontalPage_Swift/ViewController.swift
//
//  ViewController.swift
//  LJX_HorizontalPage_Swift
//
//  Created by 理享学 on 2021/9/6.
//

import UIKit

let SCREEN_WIDTH = UIScreen.main.bounds.width
let SCREEN_HEIGHT = UIScreen.main.bounds.height

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
       
        view.addSubview(collectionView)
    }

    private lazy var collectionView : CollectionView = {
        let collectionView = CollectionView.init(frame: CGRect(x: 0, y: 0, width:SCREEN_WIDTH , height:SCREEN_HEIGHT))
        return collectionView
    }()

}

